src/persister.ts

```diff
@@ -4,6 +4,7 @@ interface Persister {
     exists(key: string): boolean;
 }
 
+// *****WARNING*****: DO NOT USE LocalStoragePersister for any purpose other than testing locally on browser.
```

Comment:
Not sure we can say "DO NOT USE" ... if that's really the case, lets get ride of it...

but I think the bottom line is that "We do not recommend using LocalStoragePersister." and we can say why, it's much more helpful than just telling them what to do. Discuss the tradeoff between performance and security, it's not our app, it's not our decision, but we can inform them and let them make their own decisions?